packages/common/src/types/filterGrammar.test.ts

```diff
@@ -283,4 +283,18 @@ describe('Parse metric filters', () => {
             },
         ]);
     });
+
+    it('Should parse multiple filter values using grammar', () => {
+        const filters = [{ name: ['cat', 'dog'] }];
+        expect(removeIds(parseFilters(filters))).toStrictEqual([
+            {
+                id: undefined,
+                operator: FilterOperator.INCLUDE,
```

Comment:
`FilterOperator.INCLUDE` is going to use the SQL operator `LIKE`. We should expect `FilterOperator.EQUALS`